rest/blip_client_test.go

```diff
@@ -1028,11 +1035,26 @@ func (btc *BlipTesterCollectionClient) GetVersion(docID string, docVersion DocVe
 		if data, ok := rev[docVersion.RevID]; ok && data != nil {
 			return data.body, true
 		}
+		// lookup by cv if not found using revid
+		if data, ok := rev[docVersion.CV.String()]; ok && data != nil {
+			return data.body, true
+		}
 	}
 
 	return nil, false
 }
 
+func (btc *BlipTesterClient) AssertOnBlipHistory(t *testing.T, msg *blip.Message, docVersion DocVersion) {
+	if btc.SupportedBLIPProtocols[0] == db.CBMobileReplicationV4.SubprotocolString() {
```

Comment:
This check will break when we bump subprotocol again.

Need to make `btc.SupportedBLIPProtocols` numerical so we can do `>=` or `<=` operations on it.